README.md

```diff
@@ -40,14 +44,23 @@ You can also install packages from your distribution.
 
 ## Running the game
 
-Start the server on some machine:
+Run the game using Makefile with the command,
+
+	    make server
+  
+
+Alternatively, Start the server on some machine:
 
     ./rose-server
 
 Open a browser at http://\<server-address\>:8880 to view and control the game.
 
 Start up to 2 clients:
 
+	    make client
+    
+Or alternatively,
```

Comment:
Having alternatives for everything is confusing. We should have one simple way that works for everyone.